ui/components/MesheryAdapterPlayComponent.js

```diff
@@ -52,6 +53,11 @@ import fetchAvailableAddons from './graphql/queries/AddonsStatusQuery';
 import fetchAvailableNamespaces from "./graphql/queries/NamespaceQuery";
 import ReactSelectWrapper from "./ReactSelectWrapper";
 import MesheryMetrics from "./MesheryMetrics"
+import { errorHandlerGenerator, showProgress, successHandlerGenerator } from "./ConnectionWizard/helpers/common";
```

Comment:
Same for this, use the general one instead of the one from the connection wizard.

Reply:
@MUzairS15 still working on this one?